flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/SqlExpressionTest.scala

```diff
@@ -168,6 +168,18 @@ class SqlExpressionTest extends ExpressionTestBase {
     testSqlApi("ELEMENT(ARRAY['HELLO WORLD'])", "HELLO WORLD")
   }
 
+  @Test
+  def testRand(): Unit = {
+    val random = new java.util.Random(1)
+    testSqlApi("RAND(1)", random.nextDouble().toString)
```

Comment:
This test case can only test a expression on one row. So I will add a IT case to test rand and rand_integer on multi-rows.